src/main/java/com/carrotcreative/cream/loaders/multiple/MultipleLoader.java

```diff
@@ -10,11 +10,11 @@
 public class MultipleLoader<Identifier> {
 
     /** This policy requires everything to be downloaded, or else it fails */
-    public static final int STRICT_POLICY = 1;
+    private static final int STRICT_POLICY = 1;
```

Comment:
These actually need to remain as public, as users of a MultipleLoader will be using these to create a MultipleLoader

```java
new MultipleLoader<Identifier>(MultipleLoader.STRICT_POLICY);
```